src/App.js

```diff
@@ -1,7 +1,11 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
In this component, `useEffect()` isn't used, so you can remove it from the imports '' '